frontend/userpatch.lua

```diff
@@ -0,0 +1,120 @@
+--[[--
+Allows applying developer patches while running KOReader.
+
+The contents in `koreader/userpatches/` are applied on calling `userpatch.applyPatches(priority)`.
+--]]--
+
+local isAndroid, android = pcall(require, "android")
+
+local userpatch =
+    {   -- priorities for user patches,
```

Comment:
No real need to have this different (bracket position, indentation) to what we are used to:
```lua
local userpatch = {
    -- priorities for user patches,
    early_once = "0",
    [...]
    applyPatches = function(priority) end, -- to be overwritten, if the device allows it.
}
```